### PR TITLE
refactor: improve antiaim documentation

### PR DESCRIPTION
**Pull request by Le0Developer**
- Add info box that Override* functions can only be used during the `prediction` callback
- Add value ranges
- Add example values which make (more) sense
- Change type from float to int

### CODE REVIEW

File: developers/tables/antiaim.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # antiaim
 
+{% hint style="info" %}
+Override* functions must be called during the [prediction](../other/callbacks.md) callbacks.
```
Comment (es3n1n) [inline developers/tables/antiaim.md:4]: override functions can be called only from pre-createmove callbacks (`pre_prediction`, `prediction`)

**Review by es3n1n — CHANGES_REQUESTED** [review]

File: developers/tables/antiaim.md [hunk 2]
```diff
@@ -20,10 +24,10 @@ antiaim.OverrideInverter(false)
 
 | Name | Type | Description |
 | :--- | :--- | :--- |
-| value | float | A new desync limit value |
+| value | int | A new desync limit value (0 - 58) |
```
Comment (es3n1n) [inline developers/tables/antiaim.md:27]: this value can be float as well 🤔

**Review by es3n1n — CHANGES_REQUESTED** [review]

File: developers/tables/antiaim.md (hunk 2 above)
Comment (es3n1n) [reply developers/tables/antiaim.md:27]: i mean, every value of override* functions can be float as well

Comment (Le0Developer) [reply developers/tables/antiaim.md]: Whoops. Must've done 2 changes to fix an issue in my script and thought that that was the cause.  
Just rechecked and it indeed accepts and works with floats.

**Review by es3n1n — APPROVED** [review]
Thanks

**Pull request merged**